compose.yml

```diff
@@ -1,22 +1,13 @@
 services:
-  development:
-    container_name: fcp-mpdp-backend-development
+  fcp-mpdp-backend:
```

Comment:
Fixing name so test overrides work as expected.

(Without this then the test specific environment variables weren't being set as it saw the definitions as different services)